### PR TITLE
Update README for building on CADES SHPC Condo

### PR DESCRIPTION
**Pull request by dalg24**
Main change is using `KOKKOS_ARCH` instead of adding `-arch` flag

`CMAKE_PREFIX_PATH` is in my home directory but it shouldn't prevent anyone else to configure.
You should be able to point to it if you don;'t want to install Benchmark on your own.

### CODE REVIEW

**Comment by sslattery** [comment]
`Bikeshed in config script for condo (quote paths to TPLs)` - what is `Bikeshed`?

**Comment by Rombur** [comment]
image

**Comment by dalg24** [comment]
http://bikeshed.com

I can change the commit message if you like.  I meant this is a pointless commit where I decided to quote the paths to the TPLs.

**Comment by dalg24** [comment]
> May also want to escape TRILINOS_DIR.

You mean put double quotes around it?

**Comment by aprokop** [comment]
> You mean put double quotes around it?

Sorry, yes, that's what I meant. In general, **everything** in bash should be escaped.

File: scripts/cades_shpc_condo_cmake [hunk 1]
```diff
@@ -56,4 +56,4 @@ ARGS=(
         -D DataTransferKit_ENABLE_TESTS=ON
         -D DataTransferKit_ENABLE_EXAMPLES=ON
     )
-cmake "${ARGS[@]}" "${EXTRA_ARGS[@]}" $TRILINOS_DIR
+cmake "${ARGS[@]}" "${EXTRA_ARGS[@]}" "${TRILINOS_DIR}"
```
Comment (dalg24) [inline scripts/cades_shpc_condo_cmake]: Sharing credit for all the bikeshed

**Pull request merged**